### PR TITLE
Fix permissions on pg_test scenario file

### PR DESCRIPTION
**Pull request by Xaspy**
POSTGRESQL.

Found this problem when trying to run postgres tests. Permissions on `archiving_ready_rename.sh` are not equal to other files permissions in this directory. Changed permission to 775 (like other files in that directory).

### CODE REVIEW

**Comment by usernamedt** [comment]
Thanks! Can you also include this test to the CI pipeline?

**Comment by usernamedt** [comment]
@gsmol looks like the test introduced in [this PR](https://github.com/wal-g/wal-g/pull/950) is actually failing, can you take a look?

File: internal/databases/postgres/bguploader.go [hunk 1]
```diff
@@ -220,7 +220,8 @@ func (b *BgUploader) upload(walStatusFilename string) bool {
 		return false
 	}
 
-	if err := b.uploader.ArchiveStatusManager.MarkWalUploaded(walFilename); err != nil {
+	err = b.uploader.ArchiveStatusManager.MarkWalUploaded(walFilename)
```
Comment (serprex) [inline internal/databases/postgres/bguploader.go:223]: Revert this. You're overwriting the error from `uploadWALFile`

Comment (serprex) [reply internal/databases/postgres/bguploader.go:223]: Oh I see, `uploadWALFile` has return false, so this makes sense. So if this is breaking a test the test is wrong

**Review by serprex — APPROVED** [review]

**Pull request merged**